### ISSUE TITLE
ATLAS onboarding material

### ISSUE THREAD

**Issue by stwunsch** [root-project/training#3]
Shall we add stuff like the slides + hands-on for the ATLAS onboarding as well?

**Comment by etejedor** [comment]
Yes please! Can you add the last version of the slides / notebooks you used in the last training?

**Comment by stwunsch** [comment]
Alrighty!

**Comment by stwunsch** [comment]
So I can easily add the notebooks and stuff, but how do you add the slides? Shall I add simply the pdf or a link to the google slides (which is not persistent at all)?

**Comment by etejedor** [comment]
For example, for the summer student course we include a readme file that links to the slides:

**Comment by stwunsch** [comment]
See PR #4.
